2022-03-30 11:43:27,021 INFO    MainThread:292150 [internal.py:wandb_internal():92] W&B internal server running at pid: 292150, started at: 2022-03-30 11:43:27.020877
2022-03-30 11:43:27,058 INFO    WriterThread:292150 [datastore.py:open_for_write():77] open: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/run-24qof6aw.wandb
2022-03-30 11:43:27,069 DEBUG   SenderThread:292150 [sender.py:send():235] send: header
2022-03-30 11:43:27,070 DEBUG   SenderThread:292150 [sender.py:send():235] send: run
2022-03-30 11:43:27,418 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: check_version
2022-03-30 11:43:27,420 INFO    SenderThread:292150 [dir_watcher.py:__init__():169] watching files in: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files
2022-03-30 11:43:27,420 INFO    SenderThread:292150 [sender.py:_start_run_threads():812] run started: 24qof6aw with start time 1648640602
2022-03-30 11:43:27,420 DEBUG   SenderThread:292150 [sender.py:send():235] send: summary
2022-03-30 11:43:27,420 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-03-30 11:43:27,420 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: check_version
2022-03-30 11:43:27,526 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: run_start
2022-03-30 11:43:27,530 DEBUG   HandlerThread:292150 [meta.py:__init__():37] meta init
2022-03-30 11:43:27,530 DEBUG   HandlerThread:292150 [meta.py:__init__():51] meta init done
2022-03-30 11:43:27,530 DEBUG   HandlerThread:292150 [meta.py:probe():211] probe
2022-03-30 11:43:27,548 DEBUG   HandlerThread:292150 [meta.py:_setup_git():201] setup git
2022-03-30 11:43:27,608 DEBUG   HandlerThread:292150 [meta.py:_setup_git():208] setup git done
2022-03-30 11:43:27,609 DEBUG   HandlerThread:292150 [meta.py:_save_pip():55] save pip
2022-03-30 11:43:27,610 DEBUG   HandlerThread:292150 [meta.py:_save_pip():69] save pip done
2022-03-30 11:43:27,610 DEBUG   HandlerThread:292150 [meta.py:probe():249] probe done
2022-03-30 11:43:27,623 DEBUG   SenderThread:292150 [sender.py:send():235] send: files
2022-03-30 11:43:27,623 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-metadata.json with policy now
2022-03-30 11:43:27,694 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:43:27,694 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:43:27,905 DEBUG   SenderThread:292150 [sender.py:send():235] send: telemetry
2022-03-30 11:43:27,905 DEBUG   SenderThread:292150 [sender.py:send():235] send: telemetry
2022-03-30 11:43:27,906 DEBUG   SenderThread:292150 [sender.py:send():235] send: config
2022-03-30 11:43:27,906 DEBUG   SenderThread:292150 [sender.py:send():235] send: config
2022-03-30 11:43:28,327 INFO    Thread-11 :292150 [upload_job.py:push():137] Uploaded file /tmp/tmpiyhuw7_9wandb/jmxjcv99-wandb-metadata.json
2022-03-30 11:43:28,422 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_created():217] file/dir created: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json
2022-03-30 11:43:28,422 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_created():217] file/dir created: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/requirements.txt
2022-03-30 11:43:28,423 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_created():217] file/dir created: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-metadata.json
2022-03-30 11:43:28,423 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_created():217] file/dir created: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/output.log
2022-03-30 11:43:30,421 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/output.log
2022-03-30 11:43:46,527 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:43:47,345 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:44:09,633 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:44:31,756 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:44:55,137 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:45:12,768 ERROR   SenderThread:292150 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 159, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3/dist-packages/urllib3/util/connection.py", line 61, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 996, in _validate_conn
    conn.connect()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 314, in connect
    conn = self._new_conn()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 171, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7fac3021b4c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 719, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7fac3021b4c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 140, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ubuntu/.local/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 116, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7fac3021b4c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2022-03-30 11:45:16,900 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:45:29,930 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/output.log
2022-03-30 11:45:40,501 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:45:58,961 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:46:20,228 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:46:34,656 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:34,847 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/config.yaml
2022-03-30 11:46:35,397 DEBUG   SenderThread:292150 [sender.py:send():235] send: stats
2022-03-30 11:46:35,411 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:36,703 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:37,663 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:38,253 DEBUG   SenderThread:292150 [sender.py:send():235] send: stats
2022-03-30 11:46:38,278 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:39,489 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:40,143 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:46:40,490 DEBUG   SenderThread:292150 [sender.py:send():235] send: stats
2022-03-30 11:46:40,561 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:41,353 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:51,450 DEBUG   SenderThread:292150 [sender.py:send():235] send: stats
2022-03-30 11:46:54,492 DEBUG   SenderThread:292150 [sender.py:send():235] send: summary
2022-03-30 11:46:54,810 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-03-30 11:46:55,482 DEBUG   SenderThread:292150 [sender.py:send():235] send: summary
2022-03-30 11:46:55,507 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-03-30 11:46:55,533 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json
2022-03-30 11:46:55,584 DEBUG   SenderThread:292150 [sender.py:send():235] send: summary
2022-03-30 11:46:55,587 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-03-30 11:46:55,603 DEBUG   SenderThread:292150 [sender.py:send():235] send: summary
2022-03-30 11:46:55,604 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-03-30 11:46:56,535 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json
2022-03-30 11:46:57,187 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: stop_status
2022-03-30 11:46:57,193 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: stop_status
2022-03-30 11:46:58,664 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: log_artifact
2022-03-30 11:46:58,669 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: log_artifact
2022-03-30 11:46:58,982 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: partial_history
2022-03-30 11:47:00,789 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/output.log
2022-03-30 11:47:01,501 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:47:06,569 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:47:07,090 INFO    Thread-13 :292150 [upload_job.py:push():95] Uploaded file /home/ubuntu/.cache/wandb/artifacts/obj/md5/4e/693d8996462a50db2f031bd9943dfc
2022-03-30 11:47:12,694 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:47:18,227 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:47:18,646 INFO    SenderThread:292150 [sender.py:send_request_log_artifact():976] logged artifact 24qof6aw_submission - {'id': 'QXJ0aWZhY3Q6MTA2NjE5OTAx', 'digest': '7b789b38f7e9eaf477a2fe2ecbb60d3e', 'state': 'PENDING', 'aliases': [], 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE3NTIzODE1', 'latestArtifact': None}, 'version': 'latest'}
2022-03-30 11:47:18,802 DEBUG   SenderThread:292150 [sender.py:send():235] send: telemetry
2022-03-30 11:47:19,019 DEBUG   SenderThread:292150 [sender.py:send():235] send: telemetry
2022-03-30 11:47:19,232 DEBUG   SenderThread:292150 [sender.py:send():235] send: exit
2022-03-30 11:47:19,333 INFO    SenderThread:292150 [sender.py:send_exit():371] handling exit code: 0
2022-03-30 11:47:19,391 INFO    SenderThread:292150 [sender.py:send_exit():373] handling runtime: 212
2022-03-30 11:47:20,056 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json
2022-03-30 11:47:20,061 INFO    SenderThread:292150 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-03-30 11:47:20,159 INFO    SenderThread:292150 [sender.py:send_exit():379] send defer
2022-03-30 11:47:20,439 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: poll_exit
2022-03-30 11:47:21,100 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: poll_exit
2022-03-30 11:47:21,545 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: defer
2022-03-30 11:47:21,615 INFO    Thread-7  :292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json
2022-03-30 11:47:22,553 INFO    HandlerThread:292150 [handler.py:handle_request_defer():164] handle defer: 0
2022-03-30 11:47:23,416 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: poll_exit
2022-03-30 11:47:24,357 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:47:25,250 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: poll_exit
2022-03-30 11:47:26,963 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: defer
2022-03-30 11:47:27,533 INFO    SenderThread:292150 [sender.py:send_request_defer():388] handle sender defer: 0
2022-03-30 11:47:27,788 INFO    SenderThread:292150 [sender.py:transition_state():392] send defer: 1
2022-03-30 11:47:28,973 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: poll_exit
2022-03-30 11:47:30,335 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:47:31,711 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: poll_exit
2022-03-30 11:47:36,596 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: defer
2022-03-30 11:47:37,779 INFO    HandlerThread:292150 [handler.py:handle_request_defer():164] handle defer: 1
2022-03-30 11:48:11,055 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:16,106 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:18,787 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:26,404 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:29,790 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:31,467 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:32,553 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:38,718 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:44,224 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:50,590 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:48:57,802 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:49:02,637 WARNING MainThread:292150 [internal.py:wandb_internal():155] Internal process interrupt: 1
2022-03-30 11:49:07,890 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: poll_exit
2022-03-30 11:49:33,010 DEBUG   HandlerThread:292150 [handler.py:handle_request():141] handle_request: shutdown
2022-03-30 11:49:41,362 INFO    HandlerThread:292150 [handler.py:finish():778] shutting down handler
2022-03-30 11:49:44,742 INFO    WriterThread:292150 [datastore.py:close():281] close: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/run-24qof6aw.wandb
2022-03-30 11:52:05,786 DEBUG   SenderThread:292150 [sender.py:send_request():249] send_request: defer
2022-03-30 11:52:05,806 INFO    SenderThread:292150 [sender.py:send_request_defer():388] handle sender defer: 1
2022-03-30 11:52:05,806 INFO    SenderThread:292150 [sender.py:transition_state():392] send defer: 2
2022-03-30 11:52:05,811 INFO    SenderThread:292150 [sender.py:finish():1078] shutting down sender
2022-03-30 11:52:05,813 INFO    SenderThread:292150 [dir_watcher.py:finish():283] shutting down directory watcher
2022-03-30 11:52:06,028 INFO    SenderThread:292150 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/config.yaml
2022-03-30 11:52:06,028 INFO    SenderThread:292150 [dir_watcher.py:finish():313] scan: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files
2022-03-30 11:52:06,029 INFO    SenderThread:292150 [dir_watcher.py:finish():327] scan save: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/requirements.txt requirements.txt
2022-03-30 11:52:06,029 INFO    SenderThread:292150 [dir_watcher.py:finish():327] scan save: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-metadata.json wandb-metadata.json
2022-03-30 11:52:06,030 INFO    SenderThread:292150 [dir_watcher.py:finish():327] scan save: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/config.yaml config.yaml
2022-03-30 11:52:06,031 INFO    SenderThread:292150 [dir_watcher.py:finish():327] scan save: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json wandb-summary.json
2022-03-30 11:52:06,031 INFO    SenderThread:292150 [dir_watcher.py:finish():327] scan save: /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/output.log output.log
2022-03-30 11:52:06,031 INFO    SenderThread:292150 [file_pusher.py:finish():145] shutting down file pusher
2022-03-30 11:52:06,032 INFO    SenderThread:292150 [file_pusher.py:join():150] waiting for file pusher
2022-03-30 11:52:06,790 INFO    Thread-15 :292150 [upload_job.py:push():137] Uploaded file /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/config.yaml
2022-03-30 11:52:06,798 INFO    Thread-17 :292150 [upload_job.py:push():137] Uploaded file /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/output.log
2022-03-30 11:52:06,814 INFO    Thread-14 :292150 [upload_job.py:push():137] Uploaded file /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/requirements.txt
2022-03-30 11:52:07,297 INFO    Thread-16 :292150 [upload_job.py:push():137] Uploaded file /home/ubuntu/project/ml/wandb/run-20220330_114322-24qof6aw/files/wandb-summary.json
2022-03-30 11:52:07,694 INFO    MainThread:292150 [internal.py:handle_exit():82] Internal process exited
